client/InvoiceTickets.js

```diff
@@ -10,10 +22,38 @@ Template.InvoiceTickets.helpers({
     return InvoiceTickets.byTimeRange(
       FlowRouter.getParam("filter"),
       FlowRouter.getQueryParam("sortBy"),
-      FlowRouter.getQueryParam("sortOrder")
+      FlowRouter.getQueryParam("sortOrder"),
+      Session.get("itemsLimit")
     );
   },
   formatDate: function(date){
     return moment(date).format("MM-DD-YYYY");
   },
 });
+Template.InvoiceTickets.moreResults = function() {
+    // If, once the subscription is ready, we have less rows than we
+    // asked for, we've got all the rows in the collection.
+    return !(InvoiceTickets.find().count() < Session.get("itemsLimit"));
```

Comment:
Yes, at the end of the list ```itemsLimit``` becomes 820, but the client has only 800 invoices.
Since the invoice count is lower than ```itemsLimit``` no 'loading' indicator is shown.